aws-java-sdk-core/pom.xml

```diff
@@ -101,6 +101,24 @@
       <version>${unitils.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-api-mockito</artifactId>
+      <version>1.6.5</version>
+      <scope>test</scope>
+    </dependency>
+    <dependency>
+      <groupId>org.powermock</groupId>
+      <artifactId>powermock-module-junit4</artifactId>
+      <scope>test</scope>
+      <version>1.6.5</version>
+    </dependency>
+    <dependency>
+      <groupId>com.diffblue</groupId>
+      <artifactId>deeptestutils</artifactId>
+      <scope>test</scope>
+      <version>1.7.1</version>
```

Comment:
deeptestutils is up to 1.8.0 now

Reply:
Fixed - This was done on an older relase, and I was worried the change to 1.8.0 wasn't backwards compatible.